### PR TITLE
Update French documentation

### PR DESCRIPTION
**Pull request by dborkan**
Update French documentation.  Also the text editor I use removed a few trailing spaces at the end of sentences.

### CODE REVIEW

File: api_reference.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ Field | Description
...
 `doNotStore`             | *(optional)* Whether the API is permitted to store `comment` and `context` from this request. Stored comments will be used for future research and community model building purposes to improve the API over time. We also plan to provide dashboards and automated analysis of the comments submitted, which will apply only to those stored. Defaults to **false** (request data may be stored). **Important note:** This should be set to true if data being submitted is private (i.e. not publicly accessible), or if the data submitted contains content written by someone under 13 years old.
```
Comment (mariepellat) [inline api_reference.md:262]: I'm not sure this is totally true anymore ; maybe we can say: "There is no simple way to use the API across Alpha models and experimentally supported languages."

File: api_reference.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ FLIRTATION                        | en
 
 Please refer to the rest of the documentation bellow to appropriately set the
 language field in your request. Currently, if your comments are in both English
-and another experimentally-supported language (French, German), there's
+and another experimentally-supported language (German), there's
 no simple way to use the API that works for all comments unfortunately. You can
 either (1) change how you call the API depending on the comment language, or (2)
 wait until these models become "Alpha", when they'll be handled automatically.
```
Comment (mariepellat) [inline api_reference.md:155]: Currently if you want to use the API on German and another Alpha language (e.g. French and German or English and German etc ..) there is no simple way to do that.

Comment (dborkan) [reply api_reference.md:155]: Just updated the text to use your language although I removed the " etc .."

File: api_reference.md (hunk 1 above)
Comment (dborkan) [reply api_reference.md:262]: Done

**Review by dborkan — COMMENTED** [review]
PTAL

**Comment by mariepellat** [comment]
Looks good to me !!

**Pull request merged**